SimConnectMSFS/WasmModuleUpdater.cs

```diff
@@ -293,6 +293,12 @@ static public DateTime HubHopPresetTimestamp()
             return lastModified;
         }
 
+        static public bool HubHopPresetsPresent()
+        {
+            return System.IO.File.Exists($@"{WasmEventsHubHopFolder}\{WasmEventsHubHopFileName}") &&
+                   System.IO.File.Exists($@"{WasmEventsHubHopFolder}\{WasmEventsXplaneHubHopFileName}");
```

Comment:
You know how much I hate interpolation when you really are doing Path.Combine!!!